hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java

```diff
@@ -103,7 +104,7 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
      * @param fileStore the file store
      * @return the content about the file
      */
-    private @Nullable Optional<FileContents> contentFile(
+    private Optional<FileContents> contentFile(
```

Comment:
Ouch! How could we miss this before? 🙂

I think this is a good example where wrapping the response in an `Optional` provides no value. We should consider to change it and return `@Nullable FileContents` instead.